gramps/gen/utils/place.py

```diff
@@ -27,6 +27,7 @@
 # Standard python modules
 #
 # -------------------------------------------------------------------------
+from typing import Optional
```

Comment:
And these could be `float | None`, etc., with the future import.

Reply:
I've updated the PR, to use this preferred style. Thanks.